projects/ui/package.json

```diff
@@ -12,6 +12,6 @@
     "url": "https://github.com/vmware-clarity/ng-clarity/issues"
   },
   "peerDependencies": {
-    "@cds/core": "^6.6.0"
+    "@cds/core": "^6.6.1"
```

Comment:
This and the other peer dependency update are technically breaking changes. Just to update the PR summary (and commit message) wording.

Reply:
Oh, I see it's in the title now. Probably good enough. I'd mark this as breaking, but not really critical.